src/collections/_documentation/clients/cocoa/index.md

```diff
@@ -15,10 +15,12 @@ Getting started with Sentry is a three step process:
 <!-- WIZARD -->
 ## Installation {#install}
 
-The SDK can be installed using [CocoaPods](http://cocoapods.org) or [Carthage](https://github.com/Carthage/Carthage). This is the recommended client for both Swift and Objective-C.
+The SDK can be installed using [CocoaPods](http://cocoapods.org), [Carthage](https://github.com/Carthage/Carthage) or [Swift Package Manager](https://swift.org/package-manager/). This is the recommended client for both Swift and Objective-C.
```

Comment:
Might want to throw an Oxford comma in there before “or Swift Package Manager.”

Reply:
I do :)